sdks/python/apache_beam/transforms/validate_runner_xlang_test.py

```diff
@@ -57,6 +102,15 @@ def run_prefix(self, pipeline):
       assert_that(res, equal_to(['0a', '0b']))
 
   def run_multi_input_output_with_sideinput(self, pipeline):
+    """
+    Target transform - ParDo
+    (https://beam.apache.org/documentation/programming-guide/#pardo)
+    Test scenario - Mapping elements from multiple input collections (main
+    and side) to multiple output collections (main and side)
```

Comment:
Missing full stop here and below.

Reply:
Thanks for catching that. Will make the necessary change.